doc/sphinx/addendum/type-classes.rst

```diff
@@ -604,19 +607,22 @@ Settings
 Typeclasses eauto `:=`
 ~~~~~~~~~~~~~~~~~~~~~~
 
-.. cmd:: Typeclasses eauto := {? debug} {? {| (dfs) | (bfs) } } @natural
+.. cmd:: Typeclasses eauto := {? debug } {? ( {| bfs | dfs } ) } {? @integer }
```

Comment:
The `@integer` is still documented below as a `@natural`. BTW, it doesn't seem obvious why a negative integer should be accepted here, can we restrict the syntax instead of fixing the doc in this case?

Reply:
I think the correct fix is to update the mlg, rather than hiding weak code.
I changed the `@natural` to `@integer`.

Reply:
Yes, fixing the `mlg` (as part of another PR) is what I suggest to do.